pkg/hubble/relay/observer/observer.go

```diff
@@ -25,11 +25,9 @@ func isAvailable(conn poolTypes.ClientConn) bool {
 	if conn == nil {
 		return false
 	}
-	switch conn.GetState() {
-	case connectivity.Ready, connectivity.Idle:
-		return true
-	}
-	return false
+	state := conn.GetState()
+	return state != connectivity.TransientFailure &&
+		state != connectivity.Shutdown
```

Comment:
If I understand correctly: when the state is `connectivity.Connecting` we return `false` before this patch and `true` after this patch. What is the rational behind this change?

Reply:
Because `connectivity.Connecting` IMO shouldn't be treated as unavailable. If we're actively trying to connect I think we should try to make the request, which will block until we either successfully reconnected or failed.

But I'm also oke with leaving it as it was.